content/integrations/cloudhealth.md

```diff
@@ -4,6 +4,7 @@ integration_title: Cloudhealth
 kind: integration
 git_integration_title: cloudhealth
 newhlevel: true
+description: "Help CloudHealth help you: give it per-instance metrics from Datadog."
```

Comment:
Enrich and improve CloudHealth's right sizing and reservation recommendations with Datadog metrics.

Reply:
👍